code/datums/limb.dm

```diff
@@ -415,6 +415,58 @@
 		user.lastattacked = target
 		ON_COOLDOWN(src, "limb_cooldown", COMBAT_CLICK_DELAY)
 
+
+
+/datum/limb/mouth/maneater_mouth
```

Comment:
```suggestion
/datum/limb/mouth/maneater
```

Reply:
you are going to need to change some things to in the mouth now its a subtype too

Reply:
Yeah, im currently on it, my potato is just slowly dying while compiling because i had the audacity to push the renaming without compiling it beforehand the now i got all the bots screaming at me and compile any small change on it just to make sure xD